packages/connectivity/connectivity/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.7
+
+* Migrate the plugin to use the ConnectivityMethodChannel implementation from the connectivity_platform_interface package.
```

Comment:
"Migrate the plugin to use the `ConnectivityPlatform.instance` defined in the `connectivity_platform_interface` package."

(The plugin does not use the ConnectivityMethodChannel implementation directly)